src/main/webapp/scripts/script.js

```diff
@@ -61,15 +61,15 @@ function toggleLogin() {
         .signInWithEmailAndPassword(email, password)
         .then(function onSuccess() {
           // TODO: send token to server
-          firebase.auth()
-              .currentUser.getIdToken(/* forceRefresh */ true)
-              .then(function(idToken) {
-                // Body type defaults to JSON.
-                fetch('/login', {method: 'POST', body: idToken});
-              })
-              .catch(function onFailure(error) {
-                alert('Token retrieval failed: ' + error);
-              });
+            // firebase.auth()
+            //   .currentUser.getIdToken(/* forceRefresh */ true)
+            //   .then(function(idToken) {
+            //     // Body type defaults to JSON.
+            //     fetch('/login', {method: 'POST', body: idToken});
+            //   })
+            //   .catch(function onFailure(error) {
+            //     alert('Token retrieval failed: ' + error);
+            //   }); 
```

Comment:
Just curious, why'd all this get commented out?

Reply:
There's no longer a login servlet so it probably makes more sense to send the token when sending some other request, like a POST to RecipeServlet

Reply:
Gotcha

Reply:
Let's remove the commented code.